account_invoice_user_authorization_payment/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_move_user,access.account_move_user,model_account_move,group_user_payment,1,1,1,1
```

Comment:
This permission is already granted for `group_user_payment`, no need for it:
* The group, as it comes from `account.group_account_user` comes from `account.group_account_invoice` (https://github.com/odoo/odoo/blob/14.0/addons/account/security/account_security.xml#L63).
* And `account.group_account_invoice` has full control over `account.move` model (https://github.com/odoo/odoo/blob/14.0/addons/account/security/ir.model.access.csv#L42)